README.md

```diff
@@ -265,31 +265,36 @@ compdef kubecolor=kubectl
 
 #### fish
 
-Fish completion is officially unsupported by `kubectl`, so it is unsupported by `kubecolor` as well.
+Fish completion is officially supported by `kubectl`.
+If you did not install `kubectl` with `homebrew` (`homebrew` will auto enable autocompletion), make sure the following line is in your `~/.config/fish/config.fish` file:
```

Comment:
This note about homebrew is correct, but it's too specific as some of the other package repos also set up completions.

Maybe phrase it more general like:

```suggestion
To begin with, if you don't already have `kubectl` completion working, make sure the following line is in your `~/.config/fish/config.fish` file:
```